snippets/base/models.py

```diff
@@ -848,15 +848,16 @@ class SimpleTemplate(Template):
         Icon,
         blank=True,
         null=True,
-        on_delete=models.SET_NULL,
+        on_delete=models.PROTECT,
         verbose_name='Section Title Icon',
         related_name='simple_section_icons',
-        help_text=('Section title icon. 32x32px. PNG. '
+        help_text=('Section title icon. 16x16px. PNG. '
```

Comment:
Did this change?

Reply:
No - it's always been 16x16 on the AS side. I figured maybe just a typo on this end?